gwr/gwr.py

```diff
@@ -846,6 +846,14 @@ def llnull(self):
     def null_deviance(self):
         raise NotImplementedError('Not implemented for GWR')
 
+    @cache_readonly
+    def R2(self):
+        if isinstance(self.family, Gaussian):
+            TSS = np.sum((self.y.reshape((-1,1)) - np.mean(self.y.reshape((-1,1))))**2)
```

Comment:
why are we doing the reshapes? If they're just `(n,)`, then `(n,) - (n,)` should be `(n,)`. We'd need to do a reshape if we were trying `(n,).T.dot((n,))` ,but we're just doing `(n,)**2`.

Reply:
``self.predy`` is (n,1) so I think I had done it to make them all (n,1) to the same answer. Could just set predy to (n,) instead of reshaping all of them.